src/ResourceManagement/Sql/Sql.Tests/ScenarioTests/Sql2.DataSyncScenarioTest.cs

```diff
@@ -0,0 +1,634 @@
+                Guid dbSloBasic = new Guid("dd6d99bb-f193-4ec1-86f2-43d3bccbc49c"); // Basic
+                Guid dbSloS0 = new Guid("f1173c43-91bd-4aaa-973c-54e79e15235b "); // S0
+                Guid dbSloS1 = new Guid("1b1ebd4d-d903-4baa-97f9-4ea675f5e928"); // S1
+                Guid dbSloS2 = new Guid("455330e1-00cd-488b-b5fa-177c226f28b7"); // S2
+
+                // Variables for sync group create
+                string syncGroupName = TestUtilities.GenerateName("csm-sql-smcrud-syncgroup");
+                int interval = 300;
+                ConflictResolutionPolicyType conflictResolutionPolicy = ConflictResolutionPolicyType.Memberwin;
+                string subscriptionId = sqlClient.Credentials.SubscriptionId;
+                string syncDatabaseId = string.Format("/subscriptions/{0}/resourceGroups/{1}/providers/Microsoft.Sql/servers/{2}/databases/{3}",
+                    subscriptionId, resGroupName, serverName, databaseName);
+
+                // Variables for sync member creation
+                string syncMemberName1 = TestUtilities.GenerateName("csm-sql-smcrud-sm");
+                SyncDirectionEnum syncDirection = SyncDirectionEnum.Bidirectional;
+                DatabaseTypeEnum databaseType = DatabaseTypeEnum.AzureSqlDatabase;
+                SyncMemberGeneralParameters syncMemberGeneralParameter = new SyncMemberGeneralParameters()
+                {
+                    SyncGroupName = syncGroupName,
+                    SyncMemberName = syncMemberName1
+                };
+
+                // Create the resource group.
+                resClient.ResourceGroups.CreateOrUpdate(resGroupName, new ResourceGroup()
+                {
+                    Location = resourceGroupLocation,
+                });
+
+                try
+                {
+                    //////////////////////////////////////////////////////////////////////
+                    // Create server for test
+                    var createServerResponse = sqlClient.Servers.CreateOrUpdate(resGroupName, serverName, new ServerCreateOrUpdateParameters()
+                    {
+                        Location = serverLocation,
+                        Properties = new ServerCreateOrUpdateProperties()
+                        {
+                            AdministratorLogin = adminLogin,
+                            AdministratorLoginPassword = adminPass,
+                            Version = version,
+                        }
+                    });
+
+                    TestUtilities.ValidateOperationResponse(createServerResponse, HttpStatusCode.Created);
+
+                    //////////////////////////////////////////////////////////////////////
+                    // Create database for test
+                    var createDbResponse = sqlClient.Databases.CreateOrUpdate(resGroupName, serverName, databaseName, new DatabaseCreateOrUpdateParameters()
```

Comment:
A lot of this code looks duplicated. Consider making/using helper methods.

https://github.com/Azure/azure-sdk-for-net/blob/master/src/ResourceManagement/Sql/Sql.Tests/ScenarioTests/Sql2ScenarioHelper.cs#L212